packages/block-library/src/site-logo/edit.js

```diff
@@ -154,14 +146,7 @@ const SiteLogo = ( {
 		);
 	}
 
-	let imageWidthWithinContainer;
-
-	if ( clientWidth && naturalWidth && naturalHeight ) {
-		const exceedMaxWidth = naturalWidth > clientWidth;
-		imageWidthWithinContainer = exceedMaxWidth ? clientWidth : naturalWidth;
-	}
-
-	if ( ! isResizable || ! imageWidthWithinContainer ) {
```

Comment:
I'm not sure I understood why this condition was done this way. `imageWidthWithinContainer` returns either `clientWidth` or `naturalWidth`. Which will always be a number unless something went wrong with the image loading. So a check for `! naturalWidth || ! naturalHeight` should be enough here, to make sure the image loaded.

Reply:
This goes all the way back to #18811, cc @scruffian

Reply:
I'm sorry I don't remember!

Reply:
It's fine, I think we're good here